assets/base.css

```diff
@@ -2850,6 +2850,7 @@ details-disclosure > details {
   border-radius: var(--media-radius);
   box-shadow: var(--media-shadow-horizontal-offset) var(--media-shadow-vertical-offset) var(--media-shadow-blur-radius) rgba(var(--color-shadow), var(--media-shadow-opacity));
   z-index: -1;
+  pointer-events: none;
```

Comment:
Do we still need this `z-index: -1` after we added `pointer-events: none;`?

Reply:
Yeah so it can come up below other elements 🤔 👍

Reply:
For context Eugene, the negative z-index is a necessary evil we implemented to allow various configurations of cards, media, etc to have shadows that don't overlap the adjacent siblings up and to the left of them, which is the natural outcome in CSS
<img src="https://screenshot.click/02-04-lgkio-4cwm2.png" width="300" />